examples/awesome-vault-service/aggregator/main.go

```diff
@@ -15,28 +15,58 @@ import (
 	avtaskmanager "github.com/Layr-Labs/eigensdk-go/examples/awesome-vault-service/contracts/bindings/AwesomeVaultTaskManager"
 
 	examplecommon "github.com/Layr-Labs/eigensdk-go/examples/awesome-vault-service/common"
+
+	gotoml "github.com/pelletier/go-toml"
```

Comment:
If we don't alias this, it can be referred as just `toml`:

```suggestion
	"github.com/pelletier/go-toml"
```

Reply:
Done in [087125e](https://github.com/Layr-Labs/eigensdk-go/pull/686/commits/087125efa77eff64bf75a95aee444bb7fee0b555)